code/libraries/CryptoMarketLib/CryptoMarketLib.js

```diff
@@ -13,47 +16,62 @@ function CryptoMarket(apiKey, market){
         symbol: "BTC"
     }
     
-    this.apiKey = apiKey
     /**
      * Fetches intraday data for a cryptocurrency
      * 
-     * memberof CryptoMarket
+     * @memberof CryptoMarket
      * @param {string} symbol cryptocurrency symbol, ex "BTC" (see docs link above for all options)
      * @param {callback} callback response function(err, data)
+     * 
+     * @example
+     * 
+     * var market = CryptoMarket(API_KEY);
+     * market.intraday("BTC", function(err, data){
+     *     resp.success(data)
+     * });
+     * 
      */
     function intraday(symbol, callback){
         var options = DEFAULT_OPTIONS
-        options.apiKey = this.apiKey
+        options.apiKey = apiKey
         options.function = "DIGITAL_CURRENCY_INTRADAY"
-        options.symbol = symbol ? symbol : options.symbol
-        fetch(options, callback)
+        options.symbol = symbol ? symbol : DEFAULT_OPTIONS.symbol
+        options.market = market ? market : DEFAULT_OPTIONS.market;
+        _fetch(options, callback)
     }
     
     /**
      * Fetches daily data for a cryptocurrency
      * 
-     * memberof CryptoMarket
+     * @memberof CryptoMarket
      * @param {string} symbol cryptocurrency symbol, ex "BTC" (see docs link above for all options)
      * @param {callback} callback response function(err, data)
+     * 
+     * @example
+     * var market = CryptoMarket(API_KEY);
+     * market.daily("BTC", function(err, data){
+     *     resp.success(data)
+     * });
      */
     function daily(symbol, callback){
         var options = DEFAULT_OPTIONS
-        options.apiKey = this.apiKey
+        options.apiKey = apiKey
         options.function = "DIGITAL_CURRENCY_DAILY"
-        options.symbol = symbol ? symbol : options.symbol
-        fetch(options, callback)
+        options.symbol = symbol ? symbol : DEFAULT_OPTIONS.symbol
+        options.market = market ? market : DEFAULT_OPTIONS.market;
+        _fetch(options, callback)
     }
     
-    function fetch(options, callback){
+    function _fetch(options, callback){
         var url = URL_TEMPLATE
-        for(key in options){
-            url = url.replace('{{'+key+'}}', options[key])
+        for(var key in options){
+            url = url.replace('{{'+key+'}}', options[key]);
         }
         var r = Requests()
         var requestOptions = {
             url
         }
-        log(url)
+        //log(url); //Uncomment to see the generated URL in logs
```

Comment:
un/commenting should never be a debugging technique. add a `DEBUG=false` flag at top of code service, and an if(DEBUG) block around this log